api/src/utils/listFilterSet/index.js

```diff
@@ -200,6 +206,6 @@ export default class ListFilterSet {
    * @memberof ListView
    */
   executeQuery() {
-    return this.model.findAll(this.filters);
+    return this.model.findAndCountAll(this.filters);
   }
 }
```

Comment:
Eu to meio perdido com esse arquivo, estamos usando ele ainda, certo?

Reply:
Não estamos mais usando ele. Eu não sei te explicar se isso foi de commit antigo, ou mais novo, ou porque essa mudança ta ai kkkkk.
Mas eu te confirmo que findAndCountAll é o certo.